src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -138,7 +138,10 @@ test.each([
         await page.keys(['Shift', 'Tab', 'Shift']);
       }
       await expect(page.isFocused(wrapper.findSplitPanel().findSlider().toSelector())).resolves.toBe(true);
-      await page.keys(Array(30).fill(repeatKey));
+      // send each keystroke as individual command to allow UI re-rendering between keys
+      for (const key of Array(30).fill(repeatKey)) {
+        await page.keys(key);
+      }
```

Comment:
Replaced this code, because I have got some flaky instances where the test failed to report the correct size